sdk/python/tests/compiler/testdata/volume_op.log

```diff
@@ -0,0 +1,5 @@
+[create-pvc : main] time="2020-06-09T02:51:32Z" level=error msg="Initialize script failed: exit status 2:"
+[create-pvc : main] time="2020-06-09T02:51:32Z" level=info msg="kubectl create -f /tmp/manifest.yaml -o json"
+[create-pvc : main] time="2020-06-09T02:51:32Z" level=error msg="Run kubectl command failed with: exit status 1 and error: error validating \"/tmp/manifest.yaml\": error validating data: unknown object type \"nil\" in PersistentVolumeClaim.spec.resources.requests.storage; if you choose to ignore these errors, turn validation off with --validate=false"
+[create-pvc : main] time="2020-06-09T02:51:32Z" level=error msg="Execute resource failed: exit status 1:"
```

Comment:
You need to update the service account for this. Look at the resourceOp comment.

Reply:
I did the service account update but failed on the PersistentVolumeClaim and the missing default `size` without which this error arose:

> [create-pvc : main] time="2020-06-11T03:39:25Z" level=error msg="Run kubectl command failed with: exit status 1 and error: error validating \"/tmp/manifest.yaml\": error validating data: unknown object type \"nil\" in PersistentVolumeClaim.spec.resources.requests.storage; if you choose to ignore these errors, turn validation off with --validate=false"